css/style.css

```diff
@@ -1,18 +1,311 @@
-.main {
-    display: flex;
+body {
+	box-sizing: content-box;
+	margin: 0;
+	background-color: #f9f9f9;
+	font-family: 'Roboto', sans-serif;
 }
-.navbar{
-    height: 60px;
+
+div, h4, h3, h1{
+	display: block;
+	margin: 0;
+	padding: 0;
+	background: transparent;
+}
+.link-color{
+	color:#606060;
+}
+
+.flex{
+	display: flex;
+}
+
+#menu-icon {
+	display: flex;
+	align-items: center;
+	font-size: 1.5rem;
+	padding-left: 1rem;
+}
```

Comment:
- [ ] Good job implementing your flex class, now all you need to do, is to take out all the rest of the `display: flex;` rules through your document and add them to the HTML, for example, `<i id="menu-icon" class="material-icons flex">`

Reply:
Already tried it and the flex class wasn't active on the menu-icon